StarterProject/pageobjects/common.js

```diff
@@ -0,0 +1,87 @@
+import Page from "./page.js";
+
+export default class Common extends Page {
+
+    constructor() {
+        super();    
+    }
+
+    closeWelcomeModal(){
+        let close = "div.emailPopupRefresh img";
+
+        if (this.isWelcomeModalPresent()){
+            browser.element(close).click;
+        }
+    }
```

Comment:
General comment about page objects: The element selectors should probably be completely separate from methods that implement site functionality.  the ES6-ified page object example from http://webdriver.io/guide/testrunner/pageobjects.html is a nice example of this.  It'll allow you to re-use elements in multiple methods without duplicating selectors, as well as give tests direct access to the methods if necessary